docker/test_python.sh

```diff
@@ -2,4 +2,4 @@
 
 make -s -C /PrairieLearn test-python
 # Generate coverage.xml for codecov
-cd /PrairieLearn && python3 -m coverage xml
+cd /PrairieLearn && python3 -m coverage xml -o /PrairieLearn/apps/prairielearn/python/coverage.xml
```

Comment:
Can we make this part of the `test-python` target in the primary `Makefile` instad?

Reply:
To keep this consistent, should we just add `coverage.xml` to the .gitignore, and remove the test_python and test_javascript scripts?

Reply:
It doesn't make sense to have these single line scripts that just call out to make for me,

Reply:
Removed in [e2eaf8f](https://github.com/PrairieLearn/PrairieLearn/pull/11404/commits/e2eaf8ff9ff8d0b2944fbcc43a7b8209cdb39261). Additionally, some more unused scripts were removed in [7deccb5](https://github.com/PrairieLearn/PrairieLearn/pull/11404/commits/7deccb5f721a8c7d61528aefe5e372a288c9cb6e).